Makefile

```diff
@@ -45,7 +45,7 @@ trim-dictionaries:
 	done
 
 check-manifest:
-	check-manifest
+	check-manifest --no-build-isolation
 
 check-distutils:
 	python setup.py check --restructuredtext --strict
```

Comment:
The `setup.py` below no longer contains a `check` command.  https://github.com/codespell-project/codespell/pull/2517#discussion_r988899083

Reply:
Seems to work fine, I get the same `running check` that I get on `master`:
```
$ python setup.py check
/Users/larsoner/opt/miniconda3/envs/mne/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py:108: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
  warnings.warn(msg, _BetaConfiguration)
running check
```